### PR TITLE
Feature/domain layer

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Address.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * @ORM\Column(type = "json_array", name="address_lines", nullable = true)
+     * @var array
+     */
+    protected $addressLines = array();
+
+    /**
+     * @ORM\Column(type = "string", nullable = true)
+     * @var string
+     */
+    protected $postcode;
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Address.php]: Didn't you teach me ... `protected $postcode = '';`

File: src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Questions/Section/AboutYou/Dependants.php [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    {
+        $this->friends = $friends;
+
+        return $this;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function isSiblings()
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Questions/Section/AboutYou/Dependants.php]: Would this be better as `hasSiblings`?

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Questions/Section/AboutYou/Dependants.php]: Not in the context of the question, it regarding who would depend on you so
is is probably the better choice here

File: src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Address.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Address.php]: :+1: Good spot

File: src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Questions/Section/AboutYou/Dependants.php (hunk 2 above)
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistanceBundle/Entity/Questions/Section/AboutYou/Dependants.php]: ok :smile:

**Pull request merged**